src/app/organisations/data/organisationsStorage.js

```diff
@@ -15,7 +15,8 @@ const {
   organisationAnnouncements,
   userOrganisationRequests,
   organisationRequestStatus,
-  getNextNumericId
+  getNextNumericId,
+  getNextLegacyId,
 } = require('./../../../infrastructure/repository');
```

Comment:
Have repository changed ?

Reply:
yeeah it looks like the function was there previously, but not imported
https://github.com/DFE-Digital/login.dfe.organisations/blob/develop/src/infrastructure/repository/index.js#L141